packages/ui/src/components/Button/dropdownButton.tsx

```diff
@@ -0,0 +1,106 @@
+import { Dropdown, Button as DefaultButton } from 'antd'
+import { MenuItemType  } from 'antd/lib/menu/hooks/useItems'
+import classNames from 'classnames'
+import React from 'react'
+
+import { MoreHorizontal } from '@open-condo/icons'
+
+import { ButtonProps } from './button'
+import { useItems } from './hooks/useItems'
+
+import { Either } from '../_utils/types'
+
+
+const DROPDOWN_CLASS_PREFIX = 'condo-dropdown'
```

Comment:
Are you sure you want to use this prefix? What if we'll add a Dropdown as an component?

Reply:
Changing the prefix is difficult for us
I added an extra class so they can be distinguished

Mb add dropdown another time?)